client/components/accordion/index.jsx

```diff
@@ -45,6 +45,10 @@ export default class Accordion extends Component {
 		this.props.onToggle( isExpanded );
 	};
 
+	_e2eAccordionType = () => (
+		kebabCase( this.props.title )
```

Comment:
The title is localized which means a) it's likely to change and b) dependent on the user's culture, so in English this will be different to other cultures. We require our tests to run in any culture (we use these tests for creating screenshots of various cultures). Is there a value which is referential and not localized?

Reply:
<img width="853" alt="screen shot 2017-08-29 at 10 30 36 am" src="https://user-images.githubusercontent.com/128826/29799234-9e771546-8ca5-11e7-9068-019f7a89b0ab.png">

Reply:
Missed that. Thanks for pointing me out. Will update this PR to use optional prop passed from the parent component